src/libs/blueprint/conduit_blueprint_mesh_examples.cpp

```diff
@@ -2532,6 +2532,175 @@ void spiral(index_t ndoms,
     }
 }
 
+//---------------------------------------------------------------------------//
+void venn(index_t nx,
+          index_t ny,
+          float64 radius, 
+          Node &res)
+{
+    res.reset();
+    // create a rectilinear coordset 
+    res["coordsets/coords/type"] = "rectilinear";
+    res["coordsets/coords/values/x"] = DataType::float64(nx+1);
+    res["coordsets/coords/values/y"] = DataType::float64(ny+1);
```

Comment:
For sections like these where you reuse a node prefix many times, I've found it to be useful to create a reference to that prefix to make things a bit more clean and concise (e.g. `conduit::Node& res_coords = res["coordsets/coords"];`).